pkg/manager/manager.go

```diff
@@ -231,22 +242,42 @@ func New(config *rest.Config, options Options) (Manager, error) {
 
 	stop := make(chan struct{})
 
+	var leaseDuration, renewDeadline, retryPeriod time.Duration
```

Comment:
`leastDuration, renewDeadline, retryPeriod := defaultLeastDuration, ...``
Then if clauses without else. Makes it much easier to read.